api/_utils/common/interfaces.ts

```diff
@@ -6,7 +6,9 @@ export interface Generic {
 }
 
 export interface RequestBody {
+  watering_id?: number;
   queryType: BodyQueryType;
+  timestamp?: string;
```

Comment:
Should not be optional. We need to throw when they are not there.

Reply:
Good catch. I will make it mandatory!